cmake/multiap-helpers.cmake

```diff
@@ -41,7 +41,11 @@ set(${PROJECT}_VERSION_MAJOR 1)
 set(${PROJECT}_VERSION_MINOR 4)
 set(${PROJECT}_VERSION_PATCH 0)
 set(${PROJECT}_VERSION_STRING ${${PROJECT}_VERSION_MAJOR}.${${PROJECT}_VERSION_MINOR}.${${PROJECT}_VERSION_PATCH})
-execute_process(COMMAND "date" "+%F %T" OUTPUT_STRIP_TRAILING_WHITESPACE OUTPUT_VARIABLE BUILD_DATE)
+
+if (NOT BUILD_DATE)
```

Comment:
No, the idea of the condition is to be able to override it from the command line. The problem with a build date is that when you build twice in a row, you get two different binaries because the build date is different.